src/browser.js

```diff
@@ -566,48 +570,48 @@
   }
 
   function getFormOptions() {
-    if (elems.preset.checked) {
```

Comment:
This logic was **literally** never reached because the preset checkbox did not track its checked state.

What i need help with is the logic nested here. For the moment it is just commented out as i have no idea if it should be used or not.

Reply:
> This logic was literally never reached because the preset checkbox did not track its checked state.

What's the basis for this statement? The `checked` attribute is an implicit property of DOM input objects of type checkbox.